lib/email/receiver.rb

```diff
@@ -1384,7 +1391,7 @@ def stage_sender_user(email, display_name)
     end
 
     def delete_staged_users
```

Comment:
Should probably be renamed to 

```suggestion
    def delete_created_staged_users
```